sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/csv/CSVFileFormat.scala

```diff
@@ -163,4 +163,3 @@ class CSVFileFormat extends TextBasedFileFormat with DataSourceRegister {
   }
 
 }
-
```

Comment:
I would don't remove this line in a revert.

Reply:
I'm taking this as OK to remove the line; please let me know if you meant removing the line in "changeset".

Reply:
I think @HyukjinKwon means removing this line in the "changeset".

Reply:
Not a big deal. Will do.